openapi.yaml

```diff
@@ -0,0 +1,511 @@
+            enabled:
+              type: boolean
+              example: true
+            max:
+              type: string
+              example: "5s"
+            min:
+              type: string
+              example: "20s"
+        condition:
+          type: object
+          properties:
+            catalog-services:
+              $ref: '#/components/schemas/CatalogServicesCondition'
+            services:
+              $ref: '#/components/schemas/ServicesSourceInput'
+            consul-kv:
+              $ref: '#/components/schemas/ConsulKVCondition'
+            schedule:
+              $ref: '#/components/schemas/ScheduleCondition'
+        source_input:
+          type: object
+          properties:
+            services:
+              $ref: '#/components/schemas/ServicesSourceInput'
+            consul-kv:
+              $ref: '#/components/schemas/ConsulKVSourceInput'
+      required:
+        - name
+        - source
+        - version
+      example:
+        condition:
+          - catalog-services:
+              - regexp: ".*"
+        description: ''
+        enabled: true
+        name: taskA
+        providers: []
+        services:
+          - web
+          - api
+        source: org/example/module
+        variables: {"val":"22","val2":"33"}
+        version: 1.0.0
+
+    Provider:
+      type: object
+      properties:
+        name:
+          type: string
+          example: aws
+        source:
+          type: string
+          example: hashicorp/aws
+        version:
+          type: string
+          example: 3.56.0
+        TerraformProvider:
+          type: array
+          items:
+            $ref: '#/components/schemas/TerraformProvider'
+      required:
+        - name
+        - source
+        - version
+
+    TerraformProvider:
+      type: object
+
+    CatalogServicesCondition:
+      type: object
+      properties:
+        regexp:
+          type: string
+          example: "web.*"
+        namespace:
+          type: string
+          example: "default"
+        node_meta:
+          type: object
+          additionalProperties:
+            type: string
+          example:
+            key: value
+        source_includes_var:
+          type: boolean
+          example: false
+    ConsulKVCondition:
+      type: object
+      properties:
+        path:
+          type: string
+          example: "my-key"
+        recurse:
+          type: boolean
+          example: false
+        datacenter:
+          type: string
+          example: "dc1"
+        namespace:
+          type: string
+          example: "default"
+        source_includes_var:
+          type: boolean
+          example: false
+    ScheduleCondition:
+      type: object
+      properties:
+        cron:
+          type: string
+          example: "* * * * Mon"
+
+    ServicesSourceInput:
+      type: object
+      properties:
+        regexp:
+          type: string
+          example: "web.*"
+    CatalogSourceInput:
```

Comment:
I don't believe we have catalog as an option for source input.

Reply:
yep, right, it's not included under the task.SourceInputs either, this was copy-paste error from conditions